docs/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,27 +1,195 @@
++
+....
 apiVersion: hazelcast.com/v1alpha1
 kind: Hazelcast
 metadata:
   name: hazelcast
 spec:
+  ...
+  licenseKeySecret: hazelcast-license-key
+  persistence:
+    baseDir: "/data/persistence/"
+    clusterDataRecoveryPolicy: "FullRecoveryOnly"
+    pvc:
+      accessModes: ["ReadWriteOnce"]
+      requestStorage: 20Gi
+....
++
+You can take backup manually or automatically(scheduled):
++
+....
+apiVersion: hazelcast.com/v1alpha1
+kind: HotBackup
+metadata:
+  name: hot-backup
+spec:
+  hazelcastResourceName: hazelcast
+....
++
+....
+apiVersion: hazelcast.com/v1alpha1
+kind: HotBackup
+metadata:
+  name: hot-backup
+spec:
+  hazelcastResourceName: hazelcast
+  schedule: "* 0-23/6 * * *"
+....
+* Restore data from
+https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVC]
+(via
+https://docs.hazelcast.com/hazelcast/latest/storage/persistence[Hazelcast
+Persistence])
++
+....
+apiVersion: hazelcast.com/v1alpha1
+kind: Hazelcast
+metadata:
+  name: hazelcast
+spec:
+  ...
+  licenseKeySecret: hazelcast-license-key
+  persistence:
+    baseDir: "/data/persistence/"
+    clusterDataRecoveryPolicy: "FullRecoveryOnly"
+    pvc:
+      accessModes: ["ReadWriteOnce"]
+      requestStorage: 20Gi
+....
+* Backup and Restore Hazelcast data to/from
+https://kubernetes.io/docs/concepts/storage/volumes/#hostpath[hostPath]
++
+....
+apiVersion: hazelcast.com/v1alpha1
+kind: Hazelcast
+metadata:
+  name: hazelcast
+spec:
+  ...
+  licenseKeySecret: hazelcast-license-key
+  persistence:
+    ...
+    hostPath: "/tmp/hazelcast"
   scheduling:
-    affinity:
-      nodeAffinity:
-...
-```
+    topologySpreadConstraints:
+      - maxSkew: 1
+        topologyKey: kubernetes.io/hostname
+        whenUnsatisfiable: DoNotSchedule
+        labelSelector:
+          matchLabels:
+            app.kubernetes.io/instance: hazelcast
+....
+* Auto
+https://docs.hazelcast.com/hazelcast/latest/storage/triggering-force-start[Force
+Start] support. The Platform Operator to detect failed Hazelcast members
+and automatically trigger a force-start
++
+....
+apiVersion: hazelcast.com/v1alpha1
+kind: Hazelcast
+metadata:
+  name: hazelcast
+spec:
+  ...
+  licenseKeySecret: hazelcast-license-key
+  persistence:
+    baseDir: "/data/persistence/"
+    clusterDataRecoveryPolicy: "FullRecoveryOnly"
+    autoForceStart: true
+....
+* ImagePullSecrets configuration for Hazelcast and Management Center CRs
+* Management Center can be started with existing PV claim:
++
+....
+apiVersion: hazelcast.com/v1alpha1
+kind: ManagementCenter
+metadata:
+  name: managementcenter
+spec:
+  ...
+  persistence:
+    enabled: true
+    existingVolumeClaimName: existing-pvc
+....
+* Pod scheduling support for Management Center CR.
+* `topologySpreadConstraints` scheduling support for Hazelcast and
+Management Center CRs.
 
 == Changes
 
-* Integrated link:https://docs.hazelcast.com/operator/5.1/phone-homes[Phone Home] service.
+* Upgrade Hazelcast and MC versions to 5.1.x
```

Comment:
Do we need to update this attributes to the new versions?

- https://github.com/hazelcast/hazelcast-platform-operator-docs/blob/340cef55aaad9059a0d4528284937f6735f66cc2/docs/antora.yml#L15
- https://github.com/hazelcast/hazelcast-platform-operator-docs/blob/340cef55aaad9059a0d4528284937f6735f66cc2/docs/antora.yml#L16

Reply:
They will be updated by release workflow at operator repo:
https://github.com/hazelcast/hazelcast-platform-operator/blob/main/.github/workflows/docs-release.yaml